styles/badge/badge.css

```diff
@@ -0,0 +1,66 @@
+.notification {
+    background-color: #555;
+    color: white;
+    text-decoration: none;
+    padding: 15px 26px;
+    position: relative;
+    display: inline-block;
+    border-radius: 2px;
+    margin:1rem;
+
+  }
+  
+  .notification:hover {
+    background: red;
+  }
+
+  
+  .notification .badge {
+    position: absolute;
+    top: -10px;
+    right: -10px;
+    padding: 5px 10px;
+    border-radius: 50%;
+    background: red;
+    color: white;
+  }
+
+  .red-badge {
+    background-color: rgb(182, 3, 3);
+  }
+
+  .badge-down {
+    position: absolute;
+    bottom: 0.1rem;
+    right: 0.1rem;
+    background-color: blue;
+    color: green;
+  }
+
+  .avatar-wrapper-large .green-badge {
+    
+    position: absolute;
+    bottom: 1rem;
+    right:0.1rem;
+    width: 0.9rem;
+    height: 0.9rem;
+    border-radius: 50%;
+    border-top: 2px solid white;
+    border-left: 2px solid white;
+    border: 2px solid white;
+  }
+
+  .green-badge {
+    background-color: green;
+    border-radius:50%;
+    width: 0.9rem;
+    height: 0.9rem;
+    margin:3rem;
+  }
+
```

Comment:
If you kept these spaces to come back and work then alright else remove unused spaces (Just suggestion)